code/modules/organs/organ_external.dm

```diff
@@ -137,53 +180,32 @@
 				burn = max(0, burn - can_inflict)
 		//If there are still hurties to dispense
 		if (burn || brute)
-			if (status & ORGAN_ROBOT && body_part != UPPER_TORSO && body_part != LOWER_TORSO)
-				droplimb(1) //Robot limbs just kinda fail at full damage.
-			else
-				//List organs we can pass it to
-				var/list/datum/organ/external/possible_points = list()
-				if(parent)
-					possible_points += parent
-				if(children)
-					possible_points += children
-				if(forbidden_limbs.len)
-					possible_points -= forbidden_limbs
-				if(possible_points.len)
-					//And pass the pain around
-					var/datum/organ/external/target = pick(possible_points)
-					target.take_damage(brute, burn, sharp, edge, used_weapon, forbidden_limbs + src)
+			owner.shock_stage += brute+burn
```

Comment:
Not generally a good thing as it's going to leave a lot of people confused when they stop doing damage from targeting a specific organ---it also helps contribute to permastunning because of the softcrit system we have. 

We've actually had this crop up as a bug and it confused players to no end why they couldn't cause any additional damage to a person.